data/json/items/comestibles/carnivore.json

```diff
@@ -11,6 +11,46 @@
     "vitamins": [  ],
     "flags": [ "RAW" ]
   },
+  {
+    "id": "froglegs",
+    "copy-from": "flesh",
+    "type": "COMESTIBLE",
+    "name": { "str": "pair of frog legs", "str_pl": "pairs of frog legs" },
```

Comment:
I think it does not have to be a pair, we can eat a single leg just fine.

Reply:
You could, but I'm using commercial references, and they're always sold as a pair of legs, still attached at the pelvic bone. Generally if you kill a frog, it's more likely you've speared it through the middle (at leas when you go gigging for frogs, that's what you do,) so the pair should be intact.  It also would maybe create an issue with weight and nutrition information since those are also based on a pair of legs at a certain combined weight.